### PR TITLE
Setup vagrant file for development

### PR DESCRIPTION
**Pull request by ChrisBr**

### CODE REVIEW

**Comment by ChrisBr** [comment]
@hennevogel please review!

File: bootstrap.sh [hunk 1]
```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+echo -e "\ninstalling required software packages...\n"
+zypper -q -n install update-alternatives ruby-devel make gcc gcc-c++ libxml2-devel libxslt-devel nodejs screen mariadb libmysqld-devel sqlite3-devel imagemagick
```
Comment (hennevogel) [inline bootstrap.sh]: Are you sure about those? `libxml2-devel`?

File: bootstrap.sh [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  cp config/database.yml.example config/database.yml
+  cp config/secrets.yml.example config/secrets.yml
+else
+  echo -e "\n\nWARNING: You have already configured your app in config/options.yml." 
+  echo -e "WARNING: Please make sure this configuration works in this vagrant box!\n\n" 
+fi 
+
+# Configure the database if it isn't
+if [ ! -f /vagrant/src/api/config/database.yml ] && [ -f /vagrant/src/api/config/database.yml.example ]; then
+  echo -e "\nSetting up your database from config/database.yml...\n"
```
Comment (hennevogel) [inline bootstrap.sh]: No copy of the example file?

File: bootstrap.sh (hunk 1 above)
Comment (ChrisBr) [inline bootstrap.sh]: nokogiri?

File: bootstrap.sh (hunk 2 above)
Comment (ChrisBr) [inline bootstrap.sh]: Hm, already did it in line 23 but you're right, it fits better here!

**Comment by ChrisBr** [comment]
@hennevogel updated!

File: bootstrap.sh [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+gem install bundler
+
+echo -e "\ninstalling your bundle...\n"
+su - vagrant -c "cd /vagrant/; bundle install --quiet"
+
+# Configure the app if it isn't
+if [ ! -f /vagrant/config/options.yml ] && [ -f /vagrant/config/options.yml.example ]; then
+  echo "Configuring your app in config/options.yml..." 
+  cp config/config.yml.example config/config.yml
+  cp config/secrets.yml.example config/secrets.yml
```
Comment (hennevogel) [inline bootstrap.sh]: If you're not checking if the file exists before this will overwrite the file if it exists already...

**Comment by ChrisBr** [comment]
@hennevogel updated!

**Comment by hennevogel** [comment]
Awesome. :+1:

**Pull request merged**